specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -341,13 +341,101 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Storage/locations/{location}/deletedAccounts/{deletedAccountName}": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Storage/deletedAccounts/{deletedAccountName}": {
       "get": {
         "tags": [
           "DeletedAccounts"
         ],
         "operationId": "DeletedAccounts_Get",
         "description": "Get properties of specified deleted account resource.",
+        "x-ms-examples": {
+          "DeletedAccountList": {
+            "$ref": "./examples/DeletedAccountGet.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeletedAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- properties retrieved and returned successfully for the deleted account.",
+            "schema": {
+              "$ref": "#/definitions/DeletedAccount"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
```

Comment:
This is also an indication that the response should be a list. I am not sure if we will support nextLink. But it is no harm to add it here.

Reply:
Don't add nextlink there if not necessary. The SDK API generated will be quite different.

Reply:
The response is always a list of object. In a general case, it won't be long. But just incase, I think we need to mark them as pageable. @blueww thoughts?

Reply:
If a nextlink is necessary, I am OK to add it.